crates/build-utils/src/content_reader.rs

```diff
@@ -24,7 +24,11 @@ impl ContentReader {
     /// Secret key location: GeneralStateTests/stRandom/randomStatetest0.json
     pub fn secret_key(path: PathWrapper) -> Result<Option<Value>, eyre::Error> {
         let path = blockchain_tests_to_general_state_tests_path(path);
-        let content = path.read_file_to_string()?;
+        let maybe_content = path.read_file_to_string();
```

Comment:
There might not be a corresponding file in the general state path with the format expected (only for Pyspecs cases)